README.md

```diff
@@ -121,9 +121,13 @@ import re
 @respond_to('hi', re.IGNORECASE)
 def hi(message):
     message.reply('I can understand hi or HI!')
-    # react with thumb up emoji
+    # add reaction with thumb up emoji
```

Comment:
I might make a new example... `wave`
```
@respond_to('wave')
def wave(message):
    # bot adds reaction of :wave: emoji to message
    message.react('wave')

   from time import sleep
   sleep(1)
 
    # bot removes it's reaction of :wave: emoji on message
   message.unreact('wave')
```

Reply:
@townie I'm just adding to above example, thx 👍